oshi-core/src/test/java/oshi/hardware/SensorsTest.java

```diff
@@ -1,9 +1,9 @@
 /**
  * OSHI (https://github.com/oshi/oshi)
- *
+ * <p>
```

Comment:
Good idea to include these, but this header will be overwritten by the maven license plugin when snapshot is deployed.   OK to leave these, for now.